x/staking/proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -79,6 +78,16 @@ message Description {
   string security_contact = 4;
   // details define other optional details.
   string details = 5;
+  // metadata defines extra information about the validator.
+  Metadata metadata = 6 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
+}
+
+// Metadata defines extra information about the validator.
+message Metadata {
+  option (gogoproto.equal) = true;
+
+  // profile_pic_uri defines a link to the validator profile picture.
+  string profile_pic_uri = 1;
```

Comment:
should we add more fields?
Like on those websites you can always find socials for instance:
https://www.stakingrewards.com/provider/binary-builders

Reply:
So we should probably have socials as a string array of URIs, wdyt ?

Reply:
Yeah that's nice